### PR TITLE
fix: add more ory open source projects

### PR DESCRIPTION
**Pull request by vinckr**
closes `product #247`

### CODE REVIEW

**Comment by vinckr** [comment]
Can you take a look @jfcurran ? 
It now looks like I want but I might have done some bad stuff with the title type 🙈 
Otherwise would mark it as ready for review.

**Comment by jfcurran** [comment]
i'll have a look

**Comment by jfcurran** [comment]
looks great, i'm glad you removed the buttons that were trailing nearly every entry of the project list

File: src/components/layouts/featurelist/feature-list-item.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import {
 } from '../../freestanding/utils/padding.module.css'
 
 export interface Features {
-  title: string
+  title: React.ReactElement
```
Comment (jfcurran) [inline src/components/layouts/featurelist/feature-list-item.tsx:16]:
```suggestion
  title: string | React.ReactElement
```

**Review by jfcurran — APPROVED** [review]
this would have saved you some of the changes, great stuff otherwise, thank you @vinckr

File: src/components/layouts/featurelist/feature-list-item.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ const FeatureListItem = ({ className, features }: PropType) => (
     <Container flexContainer={'column'}>
       {features.map((feature) => {
         return (
-          <Container className={cn(pb24)} justify={'start'} key={feature.title}>
+          <Container className={cn(pb24)} justify={'start'}>
```
Comment (vinckr) [inline src/components/layouts/featurelist/feature-list-item.tsx:31]:
```suggestion
          <Container className={cn(pb24)} justify={'start'} key={index}>
```
@jfcurran ?

**Comment by aeneasr** [comment]
Awesome stuff! I think we could automate this process also. Basically fetch the repos from github, check if they have a certain tag (e.g. "website") and then display them here! Just an idea though, and not needed to be done in this PR!

**Comment by vinckr** [comment]
Nice idea, should also be not too hard, we just have to make sure every repo has a proper description! 
Will make a note so we don't forget :-)

**Pull request merged**